guides/source/active_record_validations.md

```diff
@@ -830,6 +830,22 @@ class Person < ApplicationRecord
 end
 ```
 
+You can also use `on:` to define custom context.
+Custom contexts need to be triggered explicitly
+by passing name of the context to
+`valid?`, `invalid?`, `save`.
+
+```ruby
+class Person < ApplicationRecord
+  validates :email, uniqueness: true, on: :account_setup
+  validates :age, numericality: true, on: :account_setup
+end
+# person = Person.new
+```
+`person.valid?(:account_setup)` exceutes both the validations
```

Comment:
In the case of `person.valid?(:account_setup)`, will it run any other validations other than those from `account_setup` context?